internal/pkg/otel/samples/linux/logs_metrics_traces.yml

```
@@ -130,7 +128,7 @@ service:
 
     metrics/hostmetrics:
       receivers: [hostmetrics/system]
-      processors: [elasticinframetrics, resourcedetection, attributes/dataset, resource/process]
+      processors: [resourcedetection, attributes/dataset, resource/process]
       exporters: [elasticsearch/ecs]
```

```suggestion
      processors: [resourcedetection]
      exporters: [elasticsearch/otel]
```

Deleted components need to be removed from the config too.